bundle.proto

```diff
@@ -68,6 +69,11 @@ message InternalError {
   string msg = 1;
 }
 
+// Bundle dropped (e.g. because no leader upcoming)
+message DroppedBundle {
```

Comment:
curious if there could be more than 1 reason for bundle being dropped? maybe it can be more specific (NoLeader instead?)

Reply:
currently, only the no leader cause, but leaving it open to catch an future uses without having to modify proto